science/digital_rf/Portfile

```diff
@@ -5,7 +5,7 @@ PortGroup           cmake 1.1
 PortGroup           github 1.0
 PortGroup           cxx11 1.1
 
-github.setup        MITHaystack digital_rf 2.6.0
+github.setup        MITHaystack digital_rf 2.6.1
 revision            1
```

Comment:
Sorry, you seem to not have understood my point. The `revision` line should be deleted.

Reply:
Alright, I got rid of it. Sorry that snuck in there.

Reply:
NP